dotCMS/src/main/java/com/dotmarketing/portlets/fileassets/business/IFileAsset.java

```diff
@@ -2,6 +2,7 @@
 
 import java.io.File;
 import java.io.FileNotFoundException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.io.FileNotFoundException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

Reply:
done